.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+pnpm lint
+pnpm format
+git add --all
```

Comment:
Developers should definitely be responsible / know what code they're committing... this absolves them of having to think about staging their changes and ultimately causes a lot of churn in CI/CD pipelines. Think like managers saying "the build servers down again? Who commited broken code?"

I see the lint/format but still... it's important to be able to know what all is being changed.

Reply:
Wait, just read this was a `pre-commit` hook... yeh no. `pre-push` is fine but really in general these things should be run by a build server. 

But in lue of having a dedicated build server, I'll let this slide if we switch this `lint` / `format` over to a pre-push. But pre-commit is uber-aggressive and very anti-autonomy.